clients/src/main/java/org/apache/kafka/common/network/SSLTransportLayer.java

```diff
@@ -600,7 +601,7 @@ public Principal peerPrincipal() throws IOException {
         try {
             return sslEngine.getSession().getPeerPrincipal();
         } catch (SSLPeerUnverifiedException se) {
-            throw new IOException(String.format("Unable to retrieve getPeerPrincipal due to %s", se));
```

Comment:
Just for my benefit, what would this warning indicate to a Kafka user? Can they do something about it?